### ISSUE TITLE
Weird buggy walking, not quite seem to know where the player is sort of movement.

### ISSUE THREAD

**Issue by Eleshz** [TheCyberBrick/Spiders-2.0#42]
Here is a video, they can't quite move upwards correctly, they look like they are trying ---they look like they're at the top of a ladder but ran out of ladder, "these little bounces," they don't climb down from high places (5 blocks) to get me either...

https://user-images.githubusercontent.com/79333862/117742300-6f17a080-b1d2-11eb-8e78-bda26171c34e.mp4

**Comment by Eleshz** [comment]
### Is this project abandoned on GitHub?

**Comment by TheCyberBrick** [comment]
No, apologies for the late answer. Do you have Performant installed? This looks like an incompatibility that can occur together with the mod Performant.
